packages/admin-stories/src/docs/form/components/CustomFields.stories.mdx

```diff
@@ -0,0 +1,97 @@
+import { Meta, Story, Canvas, Source } from "@storybook/addon-docs";
+
+import dedent from "ts-dedent";
+
+<Meta title="Docs/Form/Components/Custom Fields" />
+
+# Custom Fields
+
+Comet Admin provides a set of FinalForm compatible form fields.
+
+## FinalFormInput
+
+`FinalFormInput` is a plain input field.
+You can use it like the HTML input element.
+
+<Canvas>
+    <Story id="stories-form-custom-fields--finalforminput" />
+</Canvas>
+
+## FinalFormSearchTextField
+
+`FinalFormSearchTextField` is a prestyled version of [FinalFormInput](#finalforminput).
+You can use it for implementing a search field.
+It doesn't have any built-in search functionality.
+
+<Canvas>
+    <Story id="stories-form-custom-fields--finalformsearchtextfield" />
+</Canvas>
+
+## FinalFormSelect
+
+`FinalFormSelect` provides a select with multiple options.
+It's possible to load the options asynchronously using the `useAsyncOptionsProps()` hook.
+It expects the value to be an object to be able to display the current value without having to load the async options.
+
+<Canvas>
+    <Story id="stories-form-custom-fields--finalformselect" />
+</Canvas>
+
+## FinalFormAutocomplete
+
+`FinalFormAutocomplete` provides a select with a search field.
+It can be used similarly to [FinalFormSelect](#finalformselect).
+
+<Canvas>
+    <Story id="stories-form-custom-fields--finalformautocomplete" />
+</Canvas>
+
+## Select
+
+`Select` is a thin wrapper around [Material UI's `Select`](https://v4.mui.com/components/selects/).
```

Comment:
Why do we need this component at all? What additional functionality do we add to the MUI select?

Reply:
@jamesricky You can probably answer this better

Reply:
We only add some default props. This could probably be done in the theme instead, then we could remove this component in `next`.

Reply:
Okay, I removed the docs for Select because it doesn't make sense to publish docs for a component that we plan to deprecate